meta-mender-st-stm32mp/templates/local.conf.append

```diff
@@ -0,0 +1,72 @@
+# =========================================================================
+# Appended fragment from meta-mender-community/meta-mender-st-stm32mp/templates
+# =========================================================================
+
+# Disable GRUB on ARM
+MENDER_FEATURES_ENABLE:append = " mender-uboot mender-image-gpt"
+MENDER_FEATURES_DISABLE:append = " mender-grub mender-image-uefi"
+
+# In file ./meta-st-stm32mp/conf/machine/include/st-machine-common-stm32mp.inc,
+# EXTRA_IMAGECMD_ext4 is defined to set a label based on the IMAGE_SUFFIX. But
+# IMAGE_SUFFIX is set to an empty string in meta-mender which results in invalid
+# arguments to the '-L' option.
+#
+# Just remove the '-L' argument
+EXTRA_IMAGECMD:ext4:remove = "-L"
+
+# st-partitions-image is a customer image format that is used by the STM32MP BSP
+# and it is not compatible with Mender image format requirements.
+#
+# Disable, as compatible image formats will be provided by meta-mender.
+IMAGE_CLASSES:remove = "st-partitions-image"
+
+# This is needed because we disable 'st-partitions-image' but this file will
+# still have the dependencies on bootfs, vendorfs and userfs images.
+WKS_FILE_DEPENDS:remove = "st-image-bootfs st-image-vendorfs st-image-userfs"
+
+# Disable flashlayout config
+#
+# This generates a set of configuration files to be used by the STM32 flash tool,
+# as the flash layout is not compatible with what we generate in Mender we can
+# disable this.
+#
+# It also produces an build error if left enabled, as we have disabled build of
+# the bootfs/vendorfs/userfs images which this configuration relies on.
+ENABLE_FLASHLAYOUT_CONFIG = "0"
+
+# STM32MP BSP defines a custom package called kernel-imagebootfs which contains
+# the dtb and kernel files
+#
+# These are originally added in mender-setup-image.inc
+MACHINE_ESSENTIAL_EXTRA_RDEPENDS:remove:mender-image:arm = "kernel-devicetree"
+MACHINE_ESSENTIAL_EXTRA_RDEPENDS:append:mender-image:arm = " kernel-imagebootfs"
+MENDER_STORAGE_TOTAL_SIZE_MB = "2048"
+
+# # Upstream PR: https://github.com/STMicroelectronics/meta-st-stm32mp/pull/11
+PREFERRED_PROVIDER_u-boot="u-boot-stm32mp"
+PREFERRED_PROVIDER_virtual/bootloader="u-boot-stm32mp"
+
+# # Add custom image class that appends STM32MP bootloader to the gptimg
+IMAGE_CLASSES += "stm32mp-gptimg"
+
+
+MENDER_BOOT_PART = "${MENDER_STORAGE_DEVICE_BASE}6"
```

Comment:
these numbers need to be checked once the custom board leftovers are removed

Reply:
this one ok. according gptimg class